desktop/core/src/desktop/conf.py

```diff
@@ -196,6 +196,19 @@ def is_custom_jwt_auth_enabled():
   type=int,
   default=1)
 
+GUNICORN_WORKER_TIMEOUT = Config(
+  key="gunicorn_work_timeout",
```

Comment:
please keep the full name `gunicorn_work_timeout` -> `gunicorn_worker_timeout`. Even though it gets longer, it keeps the meaning correct and also consistent with the internal names.

Reply:
+1